util/gh-pages/script.js

```diff
@@ -68,6 +68,24 @@
                 }
             }
         })
+        .directive('settingsDropdown', function ($document) {
+            return {
+                restrict: 'A',
+                link: function ($scope, $element, $attr) {
+                    $element.bind('click', function () {
+                        $element.toggleClass('open');
+                        $element.addClass('open-recent');
+                    });
+
+                    $document.bind('click', function () {
+                        if (!$element.hasClass('open-recent')) {
+                            $element.removeClass('open');
+                        }
+                        $element.removeClass('open-recent');
+                    })
+                }
+            }
+        })
```

Comment:
This and `themeDropdown` are unused now, but I think the dropdown should use this directive over the blur based one so it behaves the same as the other dropdowns on the page

Currently clicking on the `Disable keyboard shortcuts` label or inside the box but not on a control closes the popup

Reply:
> This and `themeDropdown` are unused now, but I think the dropdown should use this directive over the blur based one so it behaves the same as the other dropdowns on the page

I'm planning to make them follow the same logic (not using vue/angular) because they make the JS very difficult to debug to understand what's going on. It also allows to have less code and less JS events.

> Currently clicking on the `Disable keyboard shortcuts` label or inside the box but not on a control closes the popup

Ah indeed, forgot a `tabindex="-1"` on the menu. Now it's behaving as expected, thanks!